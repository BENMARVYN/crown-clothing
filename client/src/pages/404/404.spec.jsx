import { shallow } from "enzyme";
import NoMatchPage from "./404.component";
import React from "react";

describe("Testing 404 component", () => {
    it("Should match snapshot", () => {
        expect(shallow(<NoMatchPage />)).toMatchSnapshot()
    })
})