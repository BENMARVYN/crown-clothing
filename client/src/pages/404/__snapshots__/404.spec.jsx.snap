// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing 404 component Should match snapshot 1`] = `ShallowWrapper {}`;
